crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -47,9 +51,9 @@ impl CachedComputePipelineId {
     pub const INVALID: Self = CachedComputePipelineId(usize::MAX);
 }
 
-struct CachedPipeline {
-    descriptor: PipelineDescriptor,
-    state: CachedPipelineState,
+pub struct CachedPipeline {
```

Comment:
If those become public can you please document them in the process?

Reply:
I would rather not. If I document those, it would be something like
```
/// A cached pipeline
pub struct CachedPipeline {
...
}
```

I prefer to keep them without documentation so that someone can find them easily when checking documentation coverage and add useful doc

Reply:
Ping @alice-i-cecile for opinion. I don't think it's fair for the docs team to support the documenting effort alone.

Reply:
it's probably not fair, but I don't know what to write in docs more than the type names want space instead of camel case, and I think that's a bad practice. So is it fair blocking this PR on bad/missing docs until someone suggest in review better? It's probably fairer to that person that they open a new PR rather than wait for someone to come and I copy paste their doc

Reply:
Joke is, I actually did document those already. #5600